viewer/vueapp/src/components/connections/Connections.vue

```diff
@@ -1085,6 +1189,76 @@ export default {
       let val = this.calculateNodeWeight(nl);
       return 2 + val;
     },
+    calculateNodeLabelWeight: function (n) {
+      let val = 'normal';
+      if (this.query.baselineDate !== '0') {
+        switch (n.inresult) {
+          case 2:
+            // "old" (in baseline, not in actual result set)
+            val = 'lighter';
+            break;
+          case 1:
+            // "new" (in actual, not in baseline result set)
+            val = 'bold';
+            break;
+        }
+      }
+      return val;
+    },
+    calculateNodeLabelStyle: function (n) {
+      // italicize "old" nodes (in baseline, not in actual result set)
+      return ((this.query.baselineDate !== '0') && (n.inresult === 2)) ? 'italic' : 'normal';
+    },
+    calculateNodeLabelSuffix: function (n) {
+      let val = '';
+      if (this.query.baselineDate !== '0') {
+        switch (n.inresult) {
+          case 2:
+            // "old" (in baseline, not in actual result set)
+            val = ' 🚫';
+            break;
+          case 1:
+            // "new" (in actual, not in baseline result set)
+            val = ' ✨';
+            break;
+        }
+      }
+      return val;
+    },
+    calculateNodeBaselineVisibility: function (n) {
```

Comment:
`calculateNodeBaselineVisibility` determining whether a node is visible based on node baseline result set membership